src/diffusers/loaders/lora_conversion_utils.py

```diff
@@ -663,3 +663,210 @@ def handle_qkv(sds_sd, ait_sd, sds_key, ait_keys, dims=None):
         raise ValueError(f"`old_state_dict` should be at this point but has: {list(old_state_dict.keys())}.")
 
     return new_state_dict
+
+
+def _convert_bfl_flux_control_lora_to_diffusers(old_state_dict):
```

Comment:
Okay so regarding the error you mentioned in https://github.com/huggingface/diffusers/pull/9999#issuecomment-2510934499 goes away when we use your original conversion method on-the-fly: 

<details>
<summary>Code</summary>

```py
def _convert_bfl_flux_control_lora_to_diffusers(original_state_dict):
    converted_state_dict = {}
    original_state_dict_keys = list(original_state_dict.keys())

    num_layers = 19
    num_single_layers = 38
    inner_dim = 3072
    mlp_ratio = 4.0

    def swap_scale_shift(weight):
        shift, scale = weight.chunk(2, dim=0)
        new_weight = torch.cat([scale, shift], dim=0)
        return new_weight

    for lora_key in ["lora_A", "lora_B"]:
        ## time_text_embed.timestep_embedder <-  time_in
        converted_state_dict[
            f"time_text_embed.timestep_embedder.linear_1.{lora_key}.weight"
        ] = original_state_dict.pop(f"time_in.in_layer.{lora_key}.weight")
        if f"time_in.in_layer.{lora_key}.bias" in original_state_dict_keys:
            converted_state_dict[
                f"time_text_embed.timestep_embedder.linear_1.{lora_key}.bias"
            ] = original_state_dict.pop(f"time_in.in_layer.{lora_key}.bias")

        converted_state_dict[
            f"time_text_embed.timestep_embedder.linear_2.{lora_key}.weight"
        ] = original_state_dict.pop(f"time_in.out_layer.{lora_key}.weight")
        if f"time_in.out_layer.{lora_key}.bias" in original_state_dict_keys:
            converted_state_dict[
                f"time_text_embed.timestep_embedder.linear_2.{lora_key}.bias"
            ] = original_state_dict.pop(f"time_in.out_layer.{lora_key}.bias")

        ## time_text_embed.text_embedder <- vector_in
        converted_state_dict[f"time_text_embed.text_embedder.linear_1.{lora_key}.weight"] = original_state_dict.pop(
            f"vector_in.in_layer.{lora_key}.weight"
        )
        if f"vector_in.in_layer.{lora_key}.bias" in original_state_dict_keys:
            converted_state_dict[f"time_text_embed.text_embedder.linear_1.{lora_key}.bias"] = original_state_dict.pop(
                f"vector_in.in_layer.{lora_key}.bias"
            )

        converted_state_dict[f"time_text_embed.text_embedder.linear_2.{lora_key}.weight"] = original_state_dict.pop(
            f"vector_in.out_layer.{lora_key}.weight"
        )
        if f"vector_in.out_layer.{lora_key}.bias" in original_state_dict_keys:
            converted_state_dict[f"time_text_embed.text_embedder.linear_2.{lora_key}.bias"] = original_state_dict.pop(
                f"vector_in.out_layer.{lora_key}.bias"
            )

        # guidance
        has_guidance = any("guidance" in k for k in original_state_dict)
        if has_guidance:
            converted_state_dict[
                f"time_text_embed.guidance_embedder.linear_1.{lora_key}.weight"
            ] = original_state_dict.pop(f"guidance_in.in_layer.{lora_key}.weight")
            if f"guidance_in.in_layer.{lora_key}.bias" in original_state_dict_keys:
                converted_state_dict[
                    f"time_text_embed.guidance_embedder.linear_1.{lora_key}.bias"
                ] = original_state_dict.pop(f"guidance_in.in_layer.{lora_key}.bias")

            converted_state_dict[
                f"time_text_embed.guidance_embedder.linear_2.{lora_key}.weight"
            ] = original_state_dict.pop(f"guidance_in.out_layer.{lora_key}.weight")
            if f"guidance_in.out_layer.{lora_key}.bias" in original_state_dict_keys:
                converted_state_dict[
                    f"time_text_embed.guidance_embedder.linear_2.{lora_key}.bias"
                ] = original_state_dict.pop(f"guidance_in.out_layer.{lora_key}.bias")

        # context_embedder
        converted_state_dict[f"context_embedder.{lora_key}.weight"] = original_state_dict.pop(
            f"txt_in.{lora_key}.weight"
        )
        if f"txt_in.{lora_key}.bias" in original_state_dict_keys:
            converted_state_dict[f"context_embedder.{lora_key}.bias"] = original_state_dict.pop(
                f"txt_in.{lora_key}.bias"
            )

        # x_embedder
        converted_state_dict[f"x_embedder.{lora_key}.weight"] = original_state_dict.pop(f"img_in.{lora_key}.weight")
        if f"img_in.{lora_key}.bias" in original_state_dict_keys:
            converted_state_dict[f"x_embedder.{lora_key}.bias"] = original_state_dict.pop(f"img_in.{lora_key}.bias")

    # double transformer blocks
    for i in range(num_layers):
        block_prefix = f"transformer_blocks.{i}."

        for lora_key, lora_key in zip(["lora_A", "lora_B"], ["lora_A", "lora_B"]):
            # norms
            converted_state_dict[f"{block_prefix}norm1.linear.{lora_key}.weight"] = original_state_dict.pop(
                f"double_blocks.{i}.img_mod.lin.{lora_key}.weight"
            )
            if f"double_blocks.{i}.img_mod.lin.{lora_key}.bias" in original_state_dict_keys:
                converted_state_dict[f"{block_prefix}norm1.linear.{lora_key}.bias"] = original_state_dict.pop(
                    f"double_blocks.{i}.img_mod.lin.{lora_key}.bias"
                )

            converted_state_dict[f"{block_prefix}norm1_context.linear.{lora_key}.weight"] = original_state_dict.pop(
                f"double_blocks.{i}.txt_mod.lin.{lora_key}.weight"
            )
            if f"double_blocks.{i}.txt_mod.lin.{lora_key}.bias" in original_state_dict_keys:
                converted_state_dict[f"{block_prefix}norm1_context.linear.{lora_key}.bias"] = original_state_dict.pop(
                    f"double_blocks.{i}.txt_mod.lin.{lora_key}.bias"
                )

            # Q, K, V
            if lora_key == "lora_A":
                sample_lora_weight = original_state_dict.pop(f"double_blocks.{i}.img_attn.qkv.{lora_key}.weight")
                converted_state_dict[f"{block_prefix}attn.to_v.{lora_key}.weight"] = torch.cat([sample_lora_weight])
                converted_state_dict[f"{block_prefix}attn.to_q.{lora_key}.weight"] = torch.cat([sample_lora_weight])
                converted_state_dict[f"{block_prefix}attn.to_k.{lora_key}.weight"] = torch.cat([sample_lora_weight])

                context_lora_weight = original_state_dict.pop(f"double_blocks.{i}.txt_attn.qkv.{lora_key}.weight")
                converted_state_dict[f"{block_prefix}attn.add_q_proj.{lora_key}.weight"] = torch.cat(
                    [context_lora_weight]
                )
                converted_state_dict[f"{block_prefix}attn.add_k_proj.{lora_key}.weight"] = torch.cat(
                    [context_lora_weight]
                )
                converted_state_dict[f"{block_prefix}attn.add_v_proj.{lora_key}.weight"] = torch.cat(
                    [context_lora_weight]
                )
            else:
                sample_q, sample_k, sample_v = torch.chunk(
                    original_state_dict.pop(f"double_blocks.{i}.img_attn.qkv.{lora_key}.weight"), 3, dim=0
                )
                converted_state_dict[f"{block_prefix}attn.to_q.{lora_key}.weight"] = torch.cat([sample_q])
                converted_state_dict[f"{block_prefix}attn.to_k.{lora_key}.weight"] = torch.cat([sample_k])
                converted_state_dict[f"{block_prefix}attn.to_v.{lora_key}.weight"] = torch.cat([sample_v])

                context_q, context_k, context_v = torch.chunk(
                    original_state_dict.pop(f"double_blocks.{i}.txt_attn.qkv.{lora_key}.weight"), 3, dim=0
                )
                converted_state_dict[f"{block_prefix}attn.add_q_proj.{lora_key}.weight"] = torch.cat([context_q])
                converted_state_dict[f"{block_prefix}attn.add_k_proj.{lora_key}.weight"] = torch.cat([context_k])
                converted_state_dict[f"{block_prefix}attn.add_v_proj.{lora_key}.weight"] = torch.cat([context_v])

            if f"double_blocks.{i}.img_attn.qkv.{lora_key}.bias" in original_state_dict_keys:
                sample_q_bias, sample_k_bias, sample_v_bias = torch.chunk(
                    original_state_dict.pop(f"double_blocks.{i}.img_attn.qkv.{lora_key}.bias"), 3, dim=0
                )
                converted_state_dict[f"{block_prefix}attn.to_q.{lora_key}.bias"] = torch.cat([sample_q_bias])
                converted_state_dict[f"{block_prefix}attn.to_k.{lora_key}.bias"] = torch.cat([sample_k_bias])
                converted_state_dict[f"{block_prefix}attn.to_v.{lora_key}.bias"] = torch.cat([sample_v_bias])

            if f"double_blocks.{i}.txt_attn.qkv.{lora_key}.bias" in original_state_dict_keys:
                context_q_bias, context_k_bias, context_v_bias = torch.chunk(
                    original_state_dict.pop(f"double_blocks.{i}.txt_attn.qkv.{lora_key}.bias"), 3, dim=0
                )
                converted_state_dict[f"{block_prefix}attn.add_q_proj.{lora_key}.bias"] = torch.cat([context_q_bias])
                converted_state_dict[f"{block_prefix}attn.add_k_proj.{lora_key}.bias"] = torch.cat([context_k_bias])
                converted_state_dict[f"{block_prefix}attn.add_v_proj.{lora_key}.bias"] = torch.cat([context_v_bias])

            # ff img_mlp
            converted_state_dict[f"{block_prefix}ff.net.0.proj.{lora_key}.weight"] = original_state_dict.pop(
                f"double_blocks.{i}.img_mlp.0.{lora_key}.weight"
            )
            if f"double_blocks.{i}.img_mlp.0.{lora_key}.bias" in original_state_dict_keys:
                converted_state_dict[f"{block_prefix}ff.net.0.proj.{lora_key}.bias"] = original_state_dict.pop(
                    f"double_blocks.{i}.img_mlp.0.{lora_key}.bias"
                )

            converted_state_dict[f"{block_prefix}ff.net.2.{lora_key}.weight"] = original_state_dict.pop(
                f"double_blocks.{i}.img_mlp.2.{lora_key}.weight"
            )
            if f"double_blocks.{i}.img_mlp.2.{lora_key}.bias" in original_state_dict_keys:
                converted_state_dict[f"{block_prefix}ff.net.2.{lora_key}.bias"] = original_state_dict.pop(
                    f"double_blocks.{i}.img_mlp.2.{lora_key}.bias"
                )

            converted_state_dict[f"{block_prefix}ff_context.net.0.proj.{lora_key}.weight"] = original_state_dict.pop(
                f"double_blocks.{i}.txt_mlp.0.{lora_key}.weight"
            )
            if f"double_blocks.{i}.txt_mlp.0.{lora_key}.bias" in original_state_dict_keys:
                converted_state_dict[f"{block_prefix}ff_context.net.0.proj.{lora_key}.bias"] = original_state_dict.pop(
                    f"double_blocks.{i}.txt_mlp.0.{lora_key}.bias"
                )

            converted_state_dict[f"{block_prefix}ff_context.net.2.{lora_key}.weight"] = original_state_dict.pop(
                f"double_blocks.{i}.txt_mlp.2.{lora_key}.weight"
            )
            if f"double_blocks.{i}.txt_mlp.2.{lora_key}.bias" in original_state_dict_keys:
                converted_state_dict[f"{block_prefix}ff_context.net.2.{lora_key}.bias"] = original_state_dict.pop(
                    f"double_blocks.{i}.txt_mlp.2.{lora_key}.bias"
                )

            # output projections.
            converted_state_dict[f"{block_prefix}attn.to_out.0.{lora_key}.weight"] = original_state_dict.pop(
                f"double_blocks.{i}.img_attn.proj.{lora_key}.weight"
            )
            if f"double_blocks.{i}.img_attn.proj.{lora_key}.bias" in original_state_dict_keys:
                converted_state_dict[f"{block_prefix}attn.to_out.0.{lora_key}.bias"] = original_state_dict.pop(
                    f"double_blocks.{i}.img_attn.proj.{lora_key}.bias"
                )
            converted_state_dict[f"{block_prefix}attn.to_add_out.{lora_key}.weight"] = original_state_dict.pop(
                f"double_blocks.{i}.txt_attn.proj.{lora_key}.weight"
            )
            if f"double_blocks.{i}.txt_attn.proj.{lora_key}.bias" in original_state_dict_keys:
                converted_state_dict[f"{block_prefix}attn.to_add_out.{lora_key}.bias"] = original_state_dict.pop(
                    f"double_blocks.{i}.txt_attn.proj.{lora_key}.bias"
                )

        # qk_norm
        converted_state_dict[f"{block_prefix}attn.norm_q.weight"] = original_state_dict.pop(
            f"double_blocks.{i}.img_attn.norm.query_norm.scale"
        )
        converted_state_dict[f"{block_prefix}attn.norm_k.weight"] = original_state_dict.pop(
            f"double_blocks.{i}.img_attn.norm.key_norm.scale"
        )
        converted_state_dict[f"{block_prefix}attn.norm_added_q.weight"] = original_state_dict.pop(
            f"double_blocks.{i}.txt_attn.norm.query_norm.scale"
        )
        converted_state_dict[f"{block_prefix}attn.norm_added_k.weight"] = original_state_dict.pop(
            f"double_blocks.{i}.txt_attn.norm.key_norm.scale"
        )

    # single transfomer blocks
    for i in range(num_single_layers):
        block_prefix = f"single_transformer_blocks.{i}."

        for lora_key in ["lora_A", "lora_B"]:
            # norm.linear  <- single_blocks.0.modulation.lin
            converted_state_dict[f"{block_prefix}norm.linear.{lora_key}.weight"] = original_state_dict.pop(
                f"single_blocks.{i}.modulation.lin.{lora_key}.weight"
            )
            if f"single_blocks.{i}.modulation.lin.{lora_key}.bias" in original_state_dict_keys:
                converted_state_dict[f"{block_prefix}norm.linear.{lora_key}.bias"] = original_state_dict.pop(
                    f"single_blocks.{i}.modulation.lin.{lora_key}.bias"
                )

            # Q, K, V, mlp
            mlp_hidden_dim = int(inner_dim * mlp_ratio)
            split_size = (inner_dim, inner_dim, inner_dim, mlp_hidden_dim)

            if lora_key == "lora_A":
                lora_weight = original_state_dict.pop(f"single_blocks.{i}.linear1.{lora_key}.weight")
                converted_state_dict[f"{block_prefix}attn.to_q.{lora_key}.weight"] = torch.cat([lora_weight])
                converted_state_dict[f"{block_prefix}attn.to_k.{lora_key}.weight"] = torch.cat([lora_weight])
                converted_state_dict[f"{block_prefix}attn.to_v.{lora_key}.weight"] = torch.cat([lora_weight])
                converted_state_dict[f"{block_prefix}proj_mlp.{lora_key}.weight"] = torch.cat([lora_weight])

                if f"single_blocks.{i}.linear1.{lora_key}.bias" in original_state_dict_keys:
                    lora_bias = original_state_dict.pop(f"single_blocks.{i}.linear1.{lora_key}.bias")
                    converted_state_dict[f"{block_prefix}attn.to_q.{lora_key}.bias"] = torch.cat([lora_bias])
                    converted_state_dict[f"{block_prefix}attn.to_k.{lora_key}.bias"] = torch.cat([lora_bias])
                    converted_state_dict[f"{block_prefix}attn.to_v.{lora_key}.bias"] = torch.cat([lora_bias])
                    converted_state_dict[f"{block_prefix}proj_mlp.{lora_key}.bias"] = torch.cat([lora_bias])
            else:
                q, k, v, mlp = torch.split(
                    original_state_dict.pop(f"single_blocks.{i}.linear1.{lora_key}.weight"), split_size, dim=0
                )
                converted_state_dict[f"{block_prefix}attn.to_q.{lora_key}.weight"] = torch.cat([q])
                converted_state_dict[f"{block_prefix}attn.to_k.{lora_key}.weight"] = torch.cat([k])
                converted_state_dict[f"{block_prefix}attn.to_v.{lora_key}.weight"] = torch.cat([v])
                converted_state_dict[f"{block_prefix}proj_mlp.{lora_key}.weight"] = torch.cat([mlp])

                if f"single_blocks.{i}.linear1.{lora_key}.bias" in original_state_dict_keys:
                    q_bias, k_bias, v_bias, mlp_bias = torch.split(
                        original_state_dict.pop(f"single_blocks.{i}.linear1.{lora_key}.bias"), split_size, dim=0
                    )
                    converted_state_dict[f"{block_prefix}attn.to_q.{lora_key}.bias"] = torch.cat([q_bias])
                    converted_state_dict[f"{block_prefix}attn.to_k.{lora_key}.bias"] = torch.cat([k_bias])
                    converted_state_dict[f"{block_prefix}attn.to_v.{lora_key}.bias"] = torch.cat([v_bias])
                    converted_state_dict[f"{block_prefix}proj_mlp.{lora_key}.bias"] = torch.cat([mlp_bias])

            # output projections.
            converted_state_dict[f"{block_prefix}proj_out.{lora_key}.weight"] = original_state_dict.pop(
                f"single_blocks.{i}.linear2.{lora_key}.weight"
            )
            if f"single_blocks.{i}.linear2.{lora_key}.bias" in original_state_dict_keys:
                converted_state_dict[f"{block_prefix}proj_out.{lora_key}.bias"] = original_state_dict.pop(
                    f"single_blocks.{i}.linear2.{lora_key}.bias"
                )

        # qk norm
        converted_state_dict[f"{block_prefix}attn.norm_q.weight"] = original_state_dict.pop(
            f"single_blocks.{i}.norm.query_norm.scale"
        )
        converted_state_dict[f"{block_prefix}attn.norm_k.weight"] = original_state_dict.pop(
            f"single_blocks.{i}.norm.key_norm.scale"
        )

    for lora_key in ["lora_A", "lora_B"]:
        converted_state_dict[f"proj_out.{lora_key}.weight"] = original_state_dict.pop(
            f"final_layer.linear.{lora_key}.weight"
        )
        if f"final_layer.linear.{lora_key}.bias" in original_state_dict_keys:
            converted_state_dict[f"proj_out.{lora_key}.bias"] = original_state_dict.pop(
                f"final_layer.linear.{lora_key}.bias"
            )

        converted_state_dict[f"norm_out.linear.{lora_key}.weight"] = swap_scale_shift(
            original_state_dict.pop(f"final_layer.adaLN_modulation.1.{lora_key}.weight")
        )
        if f"final_layer.adaLN_modulation.1.{lora_key}.bias" in original_state_dict_keys:
            converted_state_dict[f"norm_out.linear.{lora_key}.bias"] = swap_scale_shift(
                original_state_dict.pop(f"final_layer.adaLN_modulation.1.{lora_key}.bias")
            )

    if len(original_state_dict) > 0:
        raise ValueError(f"`original_state_dict` should be empty at this point but has {original_state_dict.keys()=}.")

    for key in list(converted_state_dict.keys()):
        converted_state_dict[f"transformer.{key}"] = converted_state_dict.pop(key)

    return converted_state_dict
```

</details>

This tells me that the new conversion method is the culprit and does also lead to quite a few unexpected keys:

<details>
<summary>Unexpected key</summary>

```
Loading adapter weights from state_dict led to unexpected keys found in the model: single_blocks.0.proj_out.lora_A.default_0.weight, single_blocks.0.proj_out.lora_B.default_0.bias, single_blocks.0.proj_out.lora_B.default_0.weight, single_blocks.0.norm.linear.lora_A.default_0.weight, single_blocks.0.norm.linear.lora_B.default_0.bias, single_blocks.0.norm.linear.lora_B.default_0.weight, single_blocks.1.proj_out.lora_A.default_0.weight, single_blocks.1.proj_out.lora_B.default_0.bias, single_blocks.1.proj_out.lora_B.default_0.weight, single_blocks.1.norm.linear.lora_A.default_0.weight, single_blocks.1.norm.linear.lora_B.default_0.bias, single_blocks.1.norm.linear.lora_B.default_0.weight, single_blocks.10.proj_out.lora_A.default_0.weight, single_blocks.10.proj_out.lora_B.default_0.bias, single_blocks.10.proj_out.lora_B.default_0.weight, single_blocks.10.norm.linear.lora_A.default_0.weight, single_blocks.10.norm.linear.lora_B.default_0.bias, single_blocks.10.norm.linear.lora_B.default_0.weight, single_blocks.11.proj_out.lora_A.default_0.weight, single_blocks.11.proj_out.lora_B.default_0.bias, single_blocks.11.proj_out.lora_B.default_0.weight, single_blocks.11.norm.linear.lora_A.default_0.weight, single_blocks.11.norm.linear.lora_B.default_0.bias, single_blocks.11.norm.linear.lora_B.default_0.weight, single_blocks.12.proj_out.lora_A.default_0.weight, single_blocks.12.proj_out.lora_B.default_0.bias, single_blocks.12.proj_out.lora_B.default_0.weight, single_blocks.12.norm.linear.lora_A.default_0.weight, single_blocks.12.norm.linear.lora_B.default_0.bias, single_blocks.12.norm.linear.lora_B.default_0.weight, single_blocks.13.proj_out.lora_A.default_0.weight, single_blocks.13.proj_out.lora_B.default_0.bias, single_blocks.13.proj_out.lora_B.default_0.weight, single_blocks.13.norm.linear.lora_A.default_0.weight, single_blocks.13.norm.linear.lora_B.default_0.bias, single_blocks.13.norm.linear.lora_B.default_0.weight, single_blocks.14.proj_out.lora_A.default_0.weight, single_blocks.14.proj_out.lora_B.default_0.bias, single_blocks.14.proj_out.lora_B.default_0.weight, single_blocks.14.norm.linear.lora_A.default_0.weight, single_blocks.14.norm.linear.lora_B.default_0.bias, single_blocks.14.norm.linear.lora_B.default_0.weight, single_blocks.15.proj_out.lora_A.default_0.weight, single_blocks.15.proj_out.lora_B.default_0.bias, single_blocks.15.proj_out.lora_B.default_0.weight, single_blocks.15.norm.linear.lora_A.default_0.weight, single_blocks.15.norm.linear.lora_B.default_0.bias, single_blocks.15.norm.linear.lora_B.default_0.weight, single_blocks.16.proj_out.lora_A.default_0.weight, single_blocks.16.proj_out.lora_B.default_0.bias, single_blocks.16.proj_out.lora_B.default_0.weight, single_blocks.16.norm.linear.lora_A.default_0.weight, single_blocks.16.norm.linear.lora_B.default_0.bias, single_blocks.16.norm.linear.lora_B.default_0.weight, single_blocks.17.proj_out.lora_A.default_0.weight, single_blocks.17.proj_out.lora_B.default_0.bias, single_blocks.17.proj_out.lora_B.default_0.weight, single_blocks.17.norm.linear.lora_A.default_0.weight, single_blocks.17.norm.linear.lora_B.default_0.bias, single_blocks.17.norm.linear.lora_B.default_0.weight, single_blocks.18.proj_out.lora_A.default_0.weight, single_blocks.18.proj_out.lora_B.default_0.bias, single_blocks.18.proj_out.lora_B.default_0.weight, single_blocks.18.norm.linear.lora_A.default_0.weight, single_blocks.18.norm.linear.lora_B.default_0.bias, single_blocks.18.norm.linear.lora_B.default_0.weight, single_blocks.19.proj_out.lora_A.default_0.weight, single_blocks.19.proj_out.lora_B.default_0.bias, single_blocks.19.proj_out.lora_B.default_0.weight, single_blocks.19.norm.linear.lora_A.default_0.weight, single_blocks.19.norm.linear.lora_B.default_0.bias, single_blocks.19.norm.linear.lora_B.default_0.weight, single_blocks.2.proj_out.lora_A.default_0.weight, single_blocks.2.proj_out.lora_B.default_0.bias, single_blocks.2.proj_out.lora_B.default_0.weight, single_blocks.2.norm.linear.lora_A.default_0.weight, single_blocks.2.norm.linear.lora_B.default_0.bias, single_blocks.2.norm.linear.lora_B.default_0.weight, single_blocks.20.proj_out.lora_A.default_0.weight, single_blocks.20.proj_out.lora_B.default_0.bias, single_blocks.20.proj_out.lora_B.default_0.weight, single_blocks.20.norm.linear.lora_A.default_0.weight, single_blocks.20.norm.linear.lora_B.default_0.bias, single_blocks.20.norm.linear.lora_B.default_0.weight, single_blocks.21.proj_out.lora_A.default_0.weight, single_blocks.21.proj_out.lora_B.default_0.bias, single_blocks.21.proj_out.lora_B.default_0.weight, single_blocks.21.norm.linear.lora_A.default_0.weight, single_blocks.21.norm.linear.lora_B.default_0.bias, single_blocks.21.norm.linear.lora_B.default_0.weight, single_blocks.22.proj_out.lora_A.default_0.weight, single_blocks.22.proj_out.lora_B.default_0.bias, single_blocks.22.proj_out.lora_B.default_0.weight, single_blocks.22.norm.linear.lora_A.default_0.weight, single_blocks.22.norm.linear.lora_B.default_0.bias, single_blocks.22.norm.linear.lora_B.default_0.weight, single_blocks.23.proj_out.lora_A.default_0.weight, single_blocks.23.proj_out.lora_B.default_0.bias, single_blocks.23.proj_out.lora_B.default_0.weight, single_blocks.23.norm.linear.lora_A.default_0.weight, single_blocks.23.norm.linear.lora_B.default_0.bias, single_blocks.23.norm.linear.lora_B.default_0.weight, single_blocks.24.proj_out.lora_A.default_0.weight, single_blocks.24.proj_out.lora_B.default_0.bias, single_blocks.24.proj_out.lora_B.default_0.weight, single_blocks.24.norm.linear.lora_A.default_0.weight, single_blocks.24.norm.linear.lora_B.default_0.bias, single_blocks.24.norm.linear.lora_B.default_0.weight, single_blocks.25.proj_out.lora_A.default_0.weight, single_blocks.25.proj_out.lora_B.default_0.bias, single_blocks.25.proj_out.lora_B.default_0.weight, single_blocks.25.norm.linear.lora_A.default_0.weight, single_blocks.25.norm.linear.lora_B.default_0.bias, single_blocks.25.norm.linear.lora_B.default_0.weight, single_blocks.26.proj_out.lora_A.default_0.weight, single_blocks.26.proj_out.lora_B.default_0.bias, single_blocks.26.proj_out.lora_B.default_0.weight, single_blocks.26.norm.linear.lora_A.default_0.weight, single_blocks.26.norm.linear.lora_B.default_0.bias, single_blocks.26.norm.linear.lora_B.default_0.weight, single_blocks.27.proj_out.lora_A.default_0.weight, single_blocks.27.proj_out.lora_B.default_0.bias, single_blocks.27.proj_out.lora_B.default_0.weight, single_blocks.27.norm.linear.lora_A.default_0.weight, single_blocks.27.norm.linear.lora_B.default_0.bias, single_blocks.27.norm.linear.lora_B.default_0.weight, single_blocks.28.proj_out.lora_A.default_0.weight, single_blocks.28.proj_out.lora_B.default_0.bias, single_blocks.28.proj_out.lora_B.default_0.weight, single_blocks.28.norm.linear.lora_A.default_0.weight, single_blocks.28.norm.linear.lora_B.default_0.bias, single_blocks.28.norm.linear.lora_B.default_0.weight, single_blocks.29.proj_out.lora_A.default_0.weight, single_blocks.29.proj_out.lora_B.default_0.bias, single_blocks.29.proj_out.lora_B.default_0.weight, single_blocks.29.norm.linear.lora_A.default_0.weight, single_blocks.29.norm.linear.lora_B.default_0.bias, single_blocks.29.norm.linear.lora_B.default_0.weight, single_blocks.3.proj_out.lora_A.default_0.weight, single_blocks.3.proj_out.lora_B.default_0.bias, single_blocks.3.proj_out.lora_B.default_0.weight, single_blocks.3.norm.linear.lora_A.default_0.weight, single_blocks.3.norm.linear.lora_B.default_0.bias, single_blocks.3.norm.linear.lora_B.default_0.weight, single_blocks.30.proj_out.lora_A.default_0.weight, single_blocks.30.proj_out.lora_B.default_0.bias, single_blocks.30.proj_out.lora_B.default_0.weight, single_blocks.30.norm.linear.lora_A.default_0.weight, single_blocks.30.norm.linear.lora_B.default_0.bias, single_blocks.30.norm.linear.lora_B.default_0.weight, single_blocks.31.proj_out.lora_A.default_0.weight, single_blocks.31.proj_out.lora_B.default_0.bias, single_blocks.31.proj_out.lora_B.default_0.weight, single_blocks.31.norm.linear.lora_A.default_0.weight, single_blocks.31.norm.linear.lora_B.default_0.bias, single_blocks.31.norm.linear.lora_B.default_0.weight, single_blocks.32.proj_out.lora_A.default_0.weight, single_blocks.32.proj_out.lora_B.default_0.bias, single_blocks.32.proj_out.lora_B.default_0.weight, single_blocks.32.norm.linear.lora_A.default_0.weight, single_blocks.32.norm.linear.lora_B.default_0.bias, single_blocks.32.norm.linear.lora_B.default_0.weight, single_blocks.33.proj_out.lora_A.default_0.weight, single_blocks.33.proj_out.lora_B.default_0.bias, single_blocks.33.proj_out.lora_B.default_0.weight, single_blocks.33.norm.linear.lora_A.default_0.weight, single_blocks.33.norm.linear.lora_B.default_0.bias, single_blocks.33.norm.linear.lora_B.default_0.weight, single_blocks.34.proj_out.lora_A.default_0.weight, single_blocks.34.proj_out.lora_B.default_0.bias, single_blocks.34.proj_out.lora_B.default_0.weight, single_blocks.34.norm.linear.lora_A.default_0.weight, single_blocks.34.norm.linear.lora_B.default_0.bias, single_blocks.34.norm.linear.lora_B.default_0.weight, single_blocks.35.proj_out.lora_A.default_0.weight, single_blocks.35.proj_out.lora_B.default_0.bias, single_blocks.35.proj_out.lora_B.default_0.weight, single_blocks.35.norm.linear.lora_A.default_0.weight, single_blocks.35.norm.linear.lora_B.default_0.bias, single_blocks.35.norm.linear.lora_B.default_0.weight, single_blocks.36.proj_out.lora_A.default_0.weight, single_blocks.36.proj_out.lora_B.default_0.bias, single_blocks.36.proj_out.lora_B.default_0.weight, single_blocks.36.norm.linear.lora_A.default_0.weight, single_blocks.36.norm.linear.lora_B.default_0.bias, single_blocks.36.norm.linear.lora_B.default_0.weight, single_blocks.37.proj_out.lora_A.default_0.weight, single_blocks.37.proj_out.lora_B.default_0.bias, single_blocks.37.proj_out.lora_B.default_0.weight, single_blocks.37.norm.linear.lora_A.default_0.weight, single_blocks.37.norm.linear.lora_B.default_0.bias, single_blocks.37.norm.linear.lora_B.default_0.weight, single_blocks.4.proj_out.lora_A.default_0.weight, single_blocks.4.proj_out.lora_B.default_0.bias, single_blocks.4.proj_out.lora_B.default_0.weight, single_blocks.4.norm.linear.lora_A.default_0.weight, single_blocks.4.norm.linear.lora_B.default_0.bias, single_blocks.4.norm.linear.lora_B.default_0.weight, single_blocks.5.proj_out.lora_A.default_0.weight, single_blocks.5.proj_out.lora_B.default_0.bias, single_blocks.5.proj_out.lora_B.default_0.weight, single_blocks.5.norm.linear.lora_A.default_0.weight, single_blocks.5.norm.linear.lora_B.default_0.bias, single_blocks.5.norm.linear.lora_B.default_0.weight, single_blocks.6.proj_out.lora_A.default_0.weight, single_blocks.6.proj_out.lora_B.default_0.bias, single_blocks.6.proj_out.lora_B.default_0.weight, single_blocks.6.norm.linear.lora_A.default_0.weight, single_blocks.6.norm.linear.lora_B.default_0.bias, single_blocks.6.norm.linear.lora_B.default_0.weight, single_blocks.7.proj_out.lora_A.default_0.weight, single_blocks.7.proj_out.lora_B.default_0.bias, single_blocks.7.proj_out.lora_B.default_0.weight, single_blocks.7.norm.linear.lora_A.default_0.weight, single_blocks.7.norm.linear.lora_B.default_0.bias, single_blocks.7.norm.linear.lora_B.default_0.weight, single_blocks.8.proj_out.lora_A.default_0.weight, single_blocks.8.proj_out.lora_B.default_0.bias, single_blocks.8.proj_out.lora_B.default_0.weight, single_blocks.8.norm.linear.lora_A.default_0.weight, single_blocks.8.norm.linear.lora_B.default_0.bias, single_blocks.8.norm.linear.lora_B.default_0.weight, single_blocks.9.proj_out.lora_A.default_0.weight, single_blocks.9.proj_out.lora_B.default_0.bias, single_blocks.9.proj_out.lora_B.default_0.weight, single_blocks.9.norm.linear.lora_A.default_0.weight, single_blocks.9.norm.linear.lora_B.default_0.bias, single_blocks.9.norm.linear.lora_B.default_0.weight, transformer_blocks.0.attn.to_add_out.0.lora_A.default_0.weight, transformer_blocks.0.attn.to_add_out.0.lora_B.default_0.bias, transformer_blocks.0.attn.to_add_out.0.lora_B.default_0.weight, transformer_blocks.0.ff.net.2.proj.lora_A.default_0.weight, transformer_blocks.0.ff.net.2.proj.lora_B.default_0.bias, transformer_blocks.0.ff.net.2.proj.lora_B.default_0.weight, transformer_blocks.0.ff_context.net.2.proj.lora_A.default_0.weight, transformer_blocks.0.ff_context.net.2.proj.lora_B.default_0.bias, transformer_blocks.0.ff_context.net.2.proj.lora_B.default_0.weight, transformer_blocks.1.attn.to_add_out.0.lora_A.default_0.weight, transformer_blocks.1.attn.to_add_out.0.lora_B.default_0.bias, transformer_blocks.1.attn.to_add_out.0.lora_B.default_0.weight, transformer_blocks.1.ff.net.2.proj.lora_A.default_0.weight, transformer_blocks.1.ff.net.2.proj.lora_B.default_0.bias, transformer_blocks.1.ff.net.2.proj.lora_B.default_0.weight, transformer_blocks.1.ff_context.net.2.proj.lora_A.default_0.weight, transformer_blocks.1.ff_context.net.2.proj.lora_B.default_0.bias, transformer_blocks.1.ff_context.net.2.proj.lora_B.default_0.weight, transformer_blocks.2.attn.to_add_out.0.lora_A.default_0.weight, transformer_blocks.2.attn.to_add_out.0.lora_B.default_0.bias, transformer_blocks.2.attn.to_add_out.0.lora_B.default_0.weight, transformer_blocks.2.ff.net.2.proj.lora_A.default_0.weight, transformer_blocks.2.ff.net.2.proj.lora_B.default_0.bias, transformer_blocks.2.ff.net.2.proj.lora_B.default_0.weight, transformer_blocks.2.ff_context.net.2.proj.lora_A.default_0.weight, transformer_blocks.2.ff_context.net.2.proj.lora_B.default_0.bias, transformer_blocks.2.ff_context.net.2.proj.lora_B.default_0.weight, transformer_blocks.3.attn.to_add_out.0.lora_A.default_0.weight, transformer_blocks.3.attn.to_add_out.0.lora_B.default_0.bias, transformer_blocks.3.attn.to_add_out.0.lora_B.default_0.weight, transformer_blocks.3.ff.net.2.proj.lora_A.default_0.weight, transformer_blocks.3.ff.net.2.proj.lora_B.default_0.bias, transformer_blocks.3.ff.net.2.proj.lora_B.default_0.weight, transformer_blocks.3.ff_context.net.2.proj.lora_A.default_0.weight, transformer_blocks.3.ff_context.net.2.proj.lora_B.default_0.bias, transformer_blocks.3.ff_context.net.2.proj.lora_B.default_0.weight, transformer_blocks.4.attn.to_add_out.0.lora_A.default_0.weight, transformer_blocks.4.attn.to_add_out.0.lora_B.default_0.bias, transformer_blocks.4.attn.to_add_out.0.lora_B.default_0.weight, transformer_blocks.4.ff.net.2.proj.lora_A.default_0.weight, transformer_blocks.4.ff.net.2.proj.lora_B.default_0.bias, transformer_blocks.4.ff.net.2.proj.lora_B.default_0.weight, transformer_blocks.4.ff_context.net.2.proj.lora_A.default_0.weight, transformer_blocks.4.ff_context.net.2.proj.lora_B.default_0.bias, transformer_blocks.4.ff_context.net.2.proj.lora_B.default_0.weight, transformer_blocks.5.attn.to_add_out.0.lora_A.default_0.weight, transformer_blocks.5.attn.to_add_out.0.lora_B.default_0.bias, transformer_blocks.5.attn.to_add_out.0.lora_B.default_0.weight, transformer_blocks.5.ff.net.2.proj.lora_A.default_0.weight, transformer_blocks.5.ff.net.2.proj.lora_B.default_0.bias, transformer_blocks.5.ff.net.2.proj.lora_B.default_0.weight, transformer_blocks.5.ff_context.net.2.proj.lora_A.default_0.weight, transformer_blocks.5.ff_context.net.2.proj.lora_B.default_0.bias, transformer_blocks.5.ff_context.net.2.proj.lora_B.default_0.weight, transformer_blocks.6.attn.to_add_out.0.lora_A.default_0.weight, transformer_blocks.6.attn.to_add_out.0.lora_B.default_0.bias, transformer_blocks.6.attn.to_add_out.0.lora_B.default_0.weight, transformer_blocks.6.ff.net.2.proj.lora_A.default_0.weight, transformer_blocks.6.ff.net.2.proj.lora_B.default_0.bias, transformer_blocks.6.ff.net.2.proj.lora_B.default_0.weight, transformer_blocks.6.ff_context.net.2.proj.lora_A.default_0.weight, transformer_blocks.6.ff_context.net.2.proj.lora_B.default_0.bias, transformer_blocks.6.ff_context.net.2.proj.lora_B.default_0.weight, transformer_blocks.7.attn.to_add_out.0.lora_A.default_0.weight, transformer_blocks.7.attn.to_add_out.0.lora_B.default_0.bias, transformer_blocks.7.attn.to_add_out.0.lora_B.default_0.weight, transformer_blocks.7.ff.net.2.proj.lora_A.default_0.weight, transformer_blocks.7.ff.net.2.proj.lora_B.default_0.bias, transformer_blocks.7.ff.net.2.proj.lora_B.default_0.weight, transformer_blocks.7.ff_context.net.2.proj.lora_A.default_0.weight, transformer_blocks.7.ff_context.net.2.proj.lora_B.default_0.bias, transformer_blocks.7.ff_context.net.2.proj.lora_B.default_0.weight, transformer_blocks.8.attn.to_add_out.0.lora_A.default_0.weight, transformer_blocks.8.attn.to_add_out.0.lora_B.default_0.bias, transformer_blocks.8.attn.to_add_out.0.lora_B.default_0.weight, transformer_blocks.8.ff.net.2.proj.lora_A.default_0.weight, transformer_blocks.8.ff.net.2.proj.lora_B.default_0.bias, transformer_blocks.8.ff.net.2.proj.lora_B.default_0.weight, transformer_blocks.8.ff_context.net.2.proj.lora_A.default_0.weight, transformer_blocks.8.ff_context.net.2.proj.lora_B.default_0.bias, transformer_blocks.8.ff_context.net.2.proj.lora_B.default_0.weight, transformer_blocks.9.attn.to_add_out.0.lora_A.default_0.weight, transformer_blocks.9.attn.to_add_out.0.lora_B.default_0.bias, transformer_blocks.9.attn.to_add_out.0.lora_B.default_0.weight, transformer_blocks.9.ff.net.2.proj.lora_A.default_0.weight, transformer_blocks.9.ff.net.2.proj.lora_B.default_0.bias, transformer_blocks.9.ff.net.2.proj.lora_B.default_0.weight, transformer_blocks.9.ff_context.net.2.proj.lora_A.default_0.weight, transformer_blocks.9.ff_context.net.2.proj.lora_B.default_0.bias, transformer_blocks.9.ff_context.net.2.proj.lora_B.default_0.weight, transformer_blocks.10.attn.to_add_out.0.lora_A.default_0.weight, transformer_blocks.10.attn.to_add_out.0.lora_B.default_0.bias, transformer_blocks.10.attn.to_add_out.0.lora_B.default_0.weight, transformer_blocks.10.ff.net.2.proj.lora_A.default_0.weight, transformer_blocks.10.ff.net.2.proj.lora_B.default_0.bias, transformer_blocks.10.ff.net.2.proj.lora_B.default_0.weight, transformer_blocks.10.ff_context.net.2.proj.lora_A.default_0.weight, transformer_blocks.10.ff_context.net.2.proj.lora_B.default_0.bias, transformer_blocks.10.ff_context.net.2.proj.lora_B.default_0.weight, transformer_blocks.11.attn.to_add_out.0.lora_A.default_0.weight, transformer_blocks.11.attn.to_add_out.0.lora_B.default_0.bias, transformer_blocks.11.attn.to_add_out.0.lora_B.default_0.weight, transformer_blocks.11.ff.net.2.proj.lora_A.default_0.weight, transformer_blocks.11.ff.net.2.proj.lora_B.default_0.bias, transformer_blocks.11.ff.net.2.proj.lora_B.default_0.weight, transformer_blocks.11.ff_context.net.2.proj.lora_A.default_0.weight, transformer_blocks.11.ff_context.net.2.proj.lora_B.default_0.bias, transformer_blocks.11.ff_context.net.2.proj.lora_B.default_0.weight, transformer_blocks.12.attn.to_add_out.0.lora_A.default_0.weight, transformer_blocks.12.attn.to_add_out.0.lora_B.default_0.bias, transformer_blocks.12.attn.to_add_out.0.lora_B.default_0.weight, transformer_blocks.12.ff.net.2.proj.lora_A.default_0.weight, transformer_blocks.12.ff.net.2.proj.lora_B.default_0.bias, transformer_blocks.12.ff.net.2.proj.lora_B.default_0.weight, transformer_blocks.12.ff_context.net.2.proj.lora_A.default_0.weight, transformer_blocks.12.ff_context.net.2.proj.lora_B.default_0.bias, transformer_blocks.12.ff_context.net.2.proj.lora_B.default_0.weight, transformer_blocks.13.attn.to_add_out.0.lora_A.default_0.weight, transformer_blocks.13.attn.to_add_out.0.lora_B.default_0.bias, transformer_blocks.13.attn.to_add_out.0.lora_B.default_0.weight, transformer_blocks.13.ff.net.2.proj.lora_A.default_0.weight, transformer_blocks.13.ff.net.2.proj.lora_B.default_0.bias, transformer_blocks.13.ff.net.2.proj.lora_B.default_0.weight, transformer_blocks.13.ff_context.net.2.proj.lora_A.default_0.weight, transformer_blocks.13.ff_context.net.2.proj.lora_B.default_0.bias, transformer_blocks.13.ff_context.net.2.proj.lora_B.default_0.weight, transformer_blocks.14.attn.to_add_out.0.lora_A.default_0.weight, transformer_blocks.14.attn.to_add_out.0.lora_B.default_0.bias, transformer_blocks.14.attn.to_add_out.0.lora_B.default_0.weight, transformer_blocks.14.ff.net.2.proj.lora_A.default_0.weight, transformer_blocks.14.ff.net.2.proj.lora_B.default_0.bias, transformer_blocks.14.ff.net.2.proj.lora_B.default_0.weight, transformer_blocks.14.ff_context.net.2.proj.lora_A.default_0.weight, transformer_blocks.14.ff_context.net.2.proj.lora_B.default_0.bias, transformer_blocks.14.ff_context.net.2.proj.lora_B.default_0.weight, transformer_blocks.15.attn.to_add_out.0.lora_A.default_0.weight, transformer_blocks.15.attn.to_add_out.0.lora_B.default_0.bias, transformer_blocks.15.attn.to_add_out.0.lora_B.default_0.weight, transformer_blocks.15.ff.net.2.proj.lora_A.default_0.weight, transformer_blocks.15.ff.net.2.proj.lora_B.default_0.bias, transformer_blocks.15.ff.net.2.proj.lora_B.default_0.weight, transformer_blocks.15.ff_context.net.2.proj.lora_A.default_0.weight, transformer_blocks.15.ff_context.net.2.proj.lora_B.default_0.bias, transformer_blocks.15.ff_context.net.2.proj.lora_B.default_0.weight, transformer_blocks.16.attn.to_add_out.0.lora_A.default_0.weight, transformer_blocks.16.attn.to_add_out.0.lora_B.default_0.bias, transformer_blocks.16.attn.to_add_out.0.lora_B.default_0.weight, transformer_blocks.16.ff.net.2.proj.lora_A.default_0.weight, transformer_blocks.16.ff.net.2.proj.lora_B.default_0.bias, transformer_blocks.16.ff.net.2.proj.lora_B.default_0.weight, transformer_blocks.16.ff_context.net.2.proj.lora_A.default_0.weight, transformer_blocks.16.ff_context.net.2.proj.lora_B.default_0.bias, transformer_blocks.16.ff_context.net.2.proj.lora_B.default_0.weight, transformer_blocks.17.attn.to_add_out.0.lora_A.default_0.weight, transformer_blocks.17.attn.to_add_out.0.lora_B.default_0.bias, transformer_blocks.17.attn.to_add_out.0.lora_B.default_0.weight, transformer_blocks.17.ff.net.2.proj.lora_A.default_0.weight, transformer_blocks.17.ff.net.2.proj.lora_B.default_0.bias, transformer_blocks.17.ff.net.2.proj.lora_B.default_0.weight, transformer_blocks.17.ff_context.net.2.proj.lora_A.default_0.weight, transformer_blocks.17.ff_context.net.2.proj.lora_B.default_0.bias, transformer_blocks.17.ff_context.net.2.proj.lora_B.default_0.weight, transformer_blocks.18.attn.to_add_out.0.lora_A.default_0.weight, transformer_blocks.18.attn.to_add_out.0.lora_B.default_0.bias, transformer_blocks.18.attn.to_add_out.0.lora_B.default_0.weight, transformer_blocks.18.ff.net.2.proj.lora_A.default_0.weight, transformer_blocks.18.ff.net.2.proj.lora_B.default_0.bias, transformer_blocks.18.ff.net.2.proj.lora_B.default_0.weight, transformer_blocks.18.ff_context.net.2.proj.lora_A.default_0.weight, transformer_blocks.18.ff_context.net.2.proj.lora_B.default_0.bias, transformer_blocks.18.ff_context.net.2.proj.lora_B.default_0.weight. Loading adapter weights from state_dict led to missing keys in the model: single_transformer_blocks.0.proj_out.lora_A.default_0.weight, single_transformer_blocks.0.proj_out.lora_B.default_0.weight, single_transformer_blocks.0.proj_out.lora_B.default_0.bias, single_transformer_blocks.1.proj_out.lora_A.default_0.weight, single_transformer_blocks.1.proj_out.lora_B.default_0.weight, single_transformer_blocks.1.proj_out.lora_B.default_0.bias, single_transformer_blocks.2.proj_out.lora_A.default_0.weight, single_transformer_blocks.2.proj_out.lora_B.default_0.weight, single_transformer_blocks.2.proj_out.lora_B.default_0.bias, single_transformer_blocks.3.proj_out.lora_A.default_0.weight, single_transformer_blocks.3.proj_out.lora_B.default_0.weight, single_transformer_blocks.3.proj_out.lora_B.default_0.bias, single_transformer_blocks.4.proj_out.lora_A.default_0.weight, single_transformer_blocks.4.proj_out.lora_B.default_0.weight, single_transformer_blocks.4.proj_out.lora_B.default_0.bias, single_transformer_blocks.5.proj_out.lora_A.default_0.weight, single_transformer_blocks.5.proj_out.lora_B.default_0.weight, single_transformer_blocks.5.proj_out.lora_B.default_0.bias, single_transformer_blocks.6.proj_out.lora_A.default_0.weight, single_transformer_blocks.6.proj_out.lora_B.default_0.weight, single_transformer_blocks.6.proj_out.lora_B.default_0.bias, single_transformer_blocks.7.proj_out.lora_A.default_0.weight, single_transformer_blocks.7.proj_out.lora_B.default_0.weight, single_transformer_blocks.7.proj_out.lora_B.default_0.bias, single_transformer_blocks.8.proj_out.lora_A.default_0.weight, single_transformer_blocks.8.proj_out.lora_B.default_0.weight, single_transformer_blocks.8.proj_out.lora_B.default_0.bias, single_transformer_blocks.9.proj_out.lora_A.default_0.weight, single_transformer_blocks.9.proj_out.lora_B.default_0.weight, single_transformer_blocks.9.proj_out.lora_B.default_0.bias, single_transformer_blocks.10.proj_out.lora_A.default_0.weight, single_transformer_blocks.10.proj_out.lora_B.default_0.weight, single_transformer_blocks.10.proj_out.lora_B.default_0.bias, single_transformer_blocks.11.proj_out.lora_A.default_0.weight, single_transformer_blocks.11.proj_out.lora_B.default_0.weight, single_transformer_blocks.11.proj_out.lora_B.default_0.bias, single_transformer_blocks.12.proj_out.lora_A.default_0.weight, single_transformer_blocks.12.proj_out.lora_B.default_0.weight, single_transformer_blocks.12.proj_out.lora_B.default_0.bias, single_transformer_blocks.13.proj_out.lora_A.default_0.weight, single_transformer_blocks.13.proj_out.lora_B.default_0.weight, single_transformer_blocks.13.proj_out.lora_B.default_0.bias, single_transformer_blocks.14.proj_out.lora_A.default_0.weight, single_transformer_blocks.14.proj_out.lora_B.default_0.weight, single_transformer_blocks.14.proj_out.lora_B.default_0.bias, single_transformer_blocks.15.proj_out.lora_A.default_0.weight, single_transformer_blocks.15.proj_out.lora_B.default_0.weight, single_transformer_blocks.15.proj_out.lora_B.default_0.bias, single_transformer_blocks.16.proj_out.lora_A.default_0.weight, single_transformer_blocks.16.proj_out.lora_B.default_0.weight, single_transformer_blocks.16.proj_out.lora_B.default_0.bias, single_transformer_blocks.17.proj_out.lora_A.default_0.weight, single_transformer_blocks.17.proj_out.lora_B.default_0.weight, single_transformer_blocks.17.proj_out.lora_B.default_0.bias, single_transformer_blocks.18.proj_out.lora_A.default_0.weight, single_transformer_blocks.18.proj_out.lora_B.default_0.weight, single_transformer_blocks.18.proj_out.lora_B.default_0.bias, single_transformer_blocks.19.proj_out.lora_A.default_0.weight, single_transformer_blocks.19.proj_out.lora_B.default_0.weight, single_transformer_blocks.19.proj_out.lora_B.default_0.bias, single_transformer_blocks.20.proj_out.lora_A.default_0.weight, single_transformer_blocks.20.proj_out.lora_B.default_0.weight, single_transformer_blocks.20.proj_out.lora_B.default_0.bias, single_transformer_blocks.21.proj_out.lora_A.default_0.weight, single_transformer_blocks.21.proj_out.lora_B.default_0.weight, single_transformer_blocks.21.proj_out.lora_B.default_0.bias, single_transformer_blocks.22.proj_out.lora_A.default_0.weight, single_transformer_blocks.22.proj_out.lora_B.default_0.weight, single_transformer_blocks.22.proj_out.lora_B.default_0.bias, single_transformer_blocks.23.proj_out.lora_A.default_0.weight, single_transformer_blocks.23.proj_out.lora_B.default_0.weight, single_transformer_blocks.23.proj_out.lora_B.default_0.bias, single_transformer_blocks.24.proj_out.lora_A.default_0.weight, single_transformer_blocks.24.proj_out.lora_B.default_0.weight, single_transformer_blocks.24.proj_out.lora_B.default_0.bias, single_transformer_blocks.25.proj_out.lora_A.default_0.weight, single_transformer_blocks.25.proj_out.lora_B.default_0.weight, single_transformer_blocks.25.proj_out.lora_B.default_0.bias, single_transformer_blocks.26.proj_out.lora_A.default_0.weight, single_transformer_blocks.26.proj_out.lora_B.default_0.weight, single_transformer_blocks.26.proj_out.lora_B.default_0.bias, single_transformer_blocks.27.proj_out.lora_A.default_0.weight, single_transformer_blocks.27.proj_out.lora_B.default_0.weight, single_transformer_blocks.27.proj_out.lora_B.default_0.bias, single_transformer_blocks.28.proj_out.lora_A.default_0.weight, single_transformer_blocks.28.proj_out.lora_B.default_0.weight, single_transformer_blocks.28.proj_out.lora_B.default_0.bias, single_transformer_blocks.29.proj_out.lora_A.default_0.weight, single_transformer_blocks.29.proj_out.lora_B.default_0.weight, single_transformer_blocks.29.proj_out.lora_B.default_0.bias, single_transformer_blocks.30.proj_out.lora_A.default_0.weight, single_transformer_blocks.30.proj_out.lora_B.default_0.weight, single_transformer_blocks.30.proj_out.lora_B.default_0.bias, single_transformer_blocks.31.proj_out.lora_A.default_0.weight, single_transformer_blocks.31.proj_out.lora_B.default_0.weight, single_transformer_blocks.31.proj_out.lora_B.default_0.bias, single_transformer_blocks.32.proj_out.lora_A.default_0.weight, single_transformer_blocks.32.proj_out.lora_B.default_0.weight, single_transformer_blocks.32.proj_out.lora_B.default_0.bias, single_transformer_blocks.33.proj_out.lora_A.default_0.weight, single_transformer_blocks.33.proj_out.lora_B.default_0.weight, single_transformer_blocks.33.proj_out.lora_B.default_0.bias, single_transformer_blocks.34.proj_out.lora_A.default_0.weight, single_transformer_blocks.34.proj_out.lora_B.default_0.weight, single_transformer_blocks.34.proj_out.lora_B.default_0.bias, single_transformer_blocks.35.proj_out.lora_A.default_0.weight, single_transformer_blocks.35.proj_out.lora_B.default_0.weight, single_transformer_blocks.35.proj_out.lora_B.default_0.bias, single_transformer_blocks.36.proj_out.lora_A.default_0.weight, single_transformer_blocks.36.proj_out.lora_B.default_0.weight, single_transformer_blocks.36.proj_out.lora_B.default_0.bias, single_transformer_blocks.37.proj_out.lora_A.default_0.weight, single_transformer_blocks.37.proj_out.lora_B.default_0.weight, single_transformer_blocks.37.proj_out.lora_B.default_0.bias.
```

</details>

Will look into this further and update.

Reply:
I tried to pinpoint the problem in the current on-the-fly conversion but couldn't do it fully. If we're unable to pinpoint it after a couple of tries, I wouldn't mind going with the original conversion script.

Reply:
I can give it a look tomorrow with fresh eyes if you don't find time today.